pkg/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+// Config is the global configuration for the training operator.
+var Config struct {
+	PyTorchInitContainerTemplateFile string
+	PyTorchInitContainerImage        string
+}
+
+const (
+	// PyTorchInitContainerImageDefault is the default image for the pytorch
+	// init container.
+	PyTorchInitContainerImageDefault = "alpine:3.10"
+	// PyTorchInitContainerTemplateFileDefault is the default template file for
+	// the pytorch init container.
+	PyTorchInitContainerTemplateFileDefault = "/etc/config/initContainer.yaml"
```

Comment:
I saw `PyTorchInitContainerImage` in the original pytorch-operator, but not `PyTorchInitContainerTemplateFileDefault`. Do we really need this one?

Reply:
The implementation in pytorch operator hard coded a path. Here we use a CLI flag to make it configurable.

Reply:
LGTM. I would suggest @Jeffwan or other pytorch-controller contributor to take a look before merge.

Reply:
/cc @Jeffwan

Reply:
Does user have any requirements to custom init container? Could we make it completely transparent to users?

Reply:
I think they may update the resource requests. WDYT

/cc @johnugeorge

Reply:
i'd like to custom the init-container-image bescause of my project worked on a network limitations env,what shou i do?